tests/integration/it_locations_test.py

```diff
@@ -0,0 +1,63 @@
+
+from timeless.restaurants.models import Location
+from timeless.companies.models import Company
+
+
+def test_create_location(db_session):
+    """Integration test for adding and selecting Location"""
+    location = create_location(db_session)
+    row = db_session.query(Location).get(location.id)
+    assert row.name == "Krusty Burger"
+
+def test_delete_location(db_session):
+    """Integration test for deleting Location"""
+    location = create_location(db_session)
+    id = location.id
+    db_session.delete(location)
+    assert not db_session.query(Location).get(location.id)
+
+def test_edit_location(db_session):
+    """Integration test for editing Location"""
+    location = create_location(db_session)
+    location.name="Another fast food restaurant"
+    location.code="NAKB"
+    location.country="France"
+    location.region="Normandy"
+    location.city="Caen"
+    location.address="Numéro de rue 2"
+    location.longitude=49
+    location.latitude=0
+    location.type="B"
+    location.status="open"
+    location.comment="An unknown fast food restaurant."
+    db_session.merge(location)
+    row = db_session.query(Location).get(location.id)
+    assert not row.name == "Krusty Burger"
+    
+
+def create_location(db_session):
```

Comment:
I'd prefer to move this function to the top of modules. Since when I were looking the first test, I didn't understand  from which place you took `create_location`. Generally it's better to declare helper functions at the top of file and then implement functions which are using them, but not in the reverse order as you did